### PR TITLE
Fixins

### PR DESCRIPTION
**Pull request by benweatherman**
- [x] Add departments
- [x] Images don’t load
- [x] Add contacts
- [x] Add dynamic content blocks
- [ ] Add hours for contacts (gonna do this in a separate PR so we can get the rest of this out there)

### CODE REVIEW

File: fixtures/contacts.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+contacts:
+    -
+        name: 311
+        email: help@austintexas.gov
+        phone:
+            default: 311
```
Comment (ifsimicoded) [inline fixtures/contacts.yaml]: data type change will require a Janis change in src/js/modules/ContactDetails and src/js/modules/ContactPhone

File: fixtures/locations.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+locations:
+    -
+        name: Austin Recycle and Reuse Drop-off Center
```
Comment (ifsimicoded) [inline fixtures/locations.yaml]: I've been passing the contact name in -- 
we'll need to update the ContactDetails module to not pass in name to the ContactAddress module
and the ContactAddress module to use location.name
src/js/modules/ContactDetails and src/js/modules/ContactAddress

File: fixtures/services/bulk-item-pickup.yaml [hunk 3]
```diff
@@ -111,6 +112,10 @@ additional_content_ar: |
         <li>السيارات المركونة.</li>
     </ul>
 
+dynamic_content:
+    -
+        type: collection_schedule
```
Comment (ifsimicoded) [inline fixtures/services/bulk-item-pickup.yaml]: the type previously was collection_schedule_block.
We'll need to update the switch statement in src/js/modules/ApplicationBlock

File: fixtures/services/hazardous-waste-dropoff.yaml [hunk 4]
```diff
@@ -54,6 +55,14 @@ additional_content_ar: |
     <p>لا تخلط العناصر. لن يتم قبول الأحمال الكبيرة للشاحنات أو المقطورة بعد الساعة 4:45 مساءً.</p>
     <p>الشركات ليست مؤهلة للتخلص من النفايات الخطرة. للحصول على معلومات حول كيفية تخلص الشركات من النفايات الخطرة، اتصل على الرقم 512-974-4336.</p>
 
+dynamic_content:
+    -
+        type: what_do_i_do_with
```
Comment (ifsimicoded) [inline fixtures/services/hazardous-waste-dropoff.yaml]: same as above -- We'll need to update the switch statement in src/js/modules/ApplicationBlock

File: fixtures/services/bulk-item-pickup.yaml (hunk 3 above)
Comment (benweatherman) [reply fixtures/services/bulk-item-pickup.yaml]: This won't change the API response

File: fixtures/services/hazardous-waste-dropoff.yaml (hunk 4 above)
Comment (benweatherman) [reply fixtures/services/hazardous-waste-dropoff.yaml]: This won't change the API response

**Pull request merged**